README.md

```diff
@@ -63,4 +63,12 @@ npm start
 ```
 This will pull in dependencies and launch the Electron application.
 
+### Frontend test
```

Comment:
I think testing is an important-enough topic for it to have it's own section in the readme. I would add a `##Testing` line above this to make a Testing section, then change your `###Frontend test` to `###Electron UI` to stay consistent with the rest of the readme.